README.md

```diff
@@ -1 +1,2 @@
-# Review-your-project-from-trials
\ No newline at end of file
+# Zunair-Yared-Microverse-Trial
+Z-Y-Trial
```

Comment:
YOU HAVE ADDED A README FILE ITS A GOOD THINK BUT I THINK IT WOULD BE BETTER
IF YOU USE PROFESSIONAL README TEMPLATE BECAUSE IT WILL CREATE IMPRESSION ON EMPLOYERS
AND USERS THAT WILL VISIT YOUR REPO 🤝